controller/api-controllers.js

```diff
@@ -42,3 +42,18 @@ exports.getArticles = (req, res, next) => {
     })
     .catch((err) => next(err));
 };
+
+exports.getCommentsById = (req, res, next) => {
+  const { article_id } = req.params;
+  const promises = [
+    selectComments(article_id),
+    checkExists("articles", "article_id", article_id),
```

Comment:
This opens up a good debate - is it worth sending through to the model when we know we'll get nothing back, or should we send back a 404 at the controller without bothering to go to the model?

Personally I'd argue that this `checkExists` function should be invoked within `selectComments` in the model. The model makes the decisions, the controller just sends back what it is given.